libs/markets/src/lib/components/market-info/tooltip-mapping.tsx

```diff
@@ -106,4 +106,47 @@ export const tooltipMapping: Record<string, ReactNode> = {
   insurancePoolFraction: t(
     'The fraction of the insurance pool balance that is carried over from the parent market to the successor.'
   ),
+  priceRange: t('The price range of the market. '),
+  commitmentMinTimeFraction: t(
+    `The minimum time fraction for a commitment. Specifies the minimum fraction of time LPs must spend 'on the book' providing their committed liquidity.`
+  ),
+  providersFeeCalculationTimeStep: t(
+    'The time step for providers fee. Specifies how often the quality of liquidity supplied by the LPS is evaluated and fees arising from that period are earmarked for specific parties.'
+  ),
+  performanceHysteresisEpochs: t(
+    'The performance hysteresis epochs. Specifies the number of liquidity epochs over which past performance will continue to affect rewards.'
+  ),
+  SLACompletionFactor: t(
+    `The SLA completion factor. Specifies the maximum fraction of their accrued fees an LP that meets the SLA implied by market.liquidity.commitmentMinTimeFraction will lose to liquidity providers that achieved a higher SLA performance than them. `
+  ),
+  bondPenaltyParameter: t(
+    'used to calculate the penalty to liquidity providers when they fail to support their open position through sufficient general+margin balance. Valid values: any decimal number >= 0 with a default value of 0.1.'
+  ),
+  nonPerformanceBondPenaltySlope: t(
+    'used to calculate how much is the LP bond slashed if they fail to reach the minimum SLA. Valid values: any decimal number >= 0 with a default value of 2.0.'
+  ),
+  nonPerformanceBondPenaltyMax: t(
```

Comment:
This one has the same description as above, is that intentional?

Reply:
https://github.com/vegaprotocol/specs/blob/cosmicelevator/protocol/0044-LIME-lp_mechanics.md 

that's how it was in the spec, but the values were different